diff --git a/lib/index.d.ts b/lib/index.d.ts
index 968c08fbbb84e9b64ac8fdede959b4958e04b2c6..9574e1d6931a8e5f9a7193bd7fd9a5ce1e3ddafc 100644
--- a/lib/index.d.ts
+++ b/lib/index.d.ts
@@ -1,9 +1,14 @@
-export default function scraper(
-  url: string
-): Promise<{
+export type Image = {
+  url: string;
+  width: string;
+  height: string;
+  "@type": "ImageObject";
+};
+
+export default function scraper(url: string): Promise<{
   url: string;
   name: string;
-  image: string;
+  image: string | Image;
   description: string;
   cookTime: string;
   prepTime: string;